daemon/vpn/quench/libquench.go

```diff
@@ -0,0 +1,272 @@
+//go:build quench
+
+package quench
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"log"
+	"net"
+	"net/netip"
+	"sync"
+
+	quenchBindigns "quench"
+
+	"github.com/NordSecurity/nordvpn-linux/daemon/vpn"
+	"github.com/NordSecurity/nordvpn-linux/internal"
+	"github.com/NordSecurity/nordvpn-linux/tunnel"
+)
+
+const (
+	vnicName            = "qtun"
+	quenchPrefix        = "[quench]"
+	quenchInterfaceAddr = "10.3.0.2/16"
+)
+
+type Logger struct{}
+
+func (l *Logger) Log(logLevel quenchBindigns.LogLevel, message string) {
+	log.Println(internal.DebugPrefix, quenchPrefix, message)
```

Comment:
Maybe the `logLevel` should also be logged, possible instead of `internal.DebugPrefix`